assets/scss/layouts/header/_header.scss

```diff
@@ -132,6 +131,16 @@
     }
 }
 
+.header-logo-position-left {
+    float: left;
+    margin-left: remCalc(40px);
+}
+
+.header-logo-position-right {
+    float: right;
+    margin-right: remCalc(40px);
+}
+
```

Comment:
instead of floating the image, change to `text-align` and add the class to `<h1 class="header-logo">`.